### DISCUSSION TITLE
Wall Placement AR not working on IOS ?

### DISCUSSION THREAD

**Discussion by rianlukog** [google/model-viewer#2538] [Q&A]
Tried to load an AR with wall placement
Does not seem to detect no matter how much I try, Is this an error of some sort ?

**Comment by elalish** [comment]
Wall placement can be tricky. Can you share a screen recording?

**Comment by ThorstenBux** [comment]
Your usdz needs to be set to wall placement. The automatic export doesn’t support it yet. I’m working on the threejs PR to allow for it.

Kind regards
Thorsten Bux

Sent from my iPhone

On 3/07/2021, at 2:59 AM, Emmett Lalish ***@***.***> wrote:

﻿

Wall placement can be tricky. Can you share a screen recording?

**Reply by elalish** [reply]
Is this a change in Quick Look? It used to be that any model can be placed on both floor and wall.

**Comment by ThorstenBux** [comment]
This is my PR, (and all I know) https://github.com/mrdoob/three.js/pull/21997

Kind regards
Thorsten Bux

Sent from my iPhone

On 3/07/2021, at 10:16 AM, Emmett Lalish ***@***.***> wrote:

﻿

Is this a change in Quick Look? It used to be that any model can be placed on both floor and wall.
